assets/templates/html5/index.html

```diff
@@ -3,13 +3,15 @@
 <head>
 	
 	<meta charset="utf-8">
-	<meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
```

Comment:
These template changes seem to break HTML5 support with OpenFL 3.6.1. Running Mode results in this:

![](https://i.imgur.com/hIP9JNq.png)